internal/data/disbursement_instructions.go

```diff
@@ -6,16 +6,54 @@ import (
 	"fmt"
 
 	"github.com/stellar/stellar-disbursement-platform-backend/db"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/utils"
 )
 
 type DisbursementInstruction struct {
-	Phone             string  `csv:"phone"`
+	Phone             *string `csv:"phone"`
+	Email             *string `csv:"email"`
```

Comment:
Using pointers as optionals is a good practice, but it doesn't require some extra work with nil checks. The full reason I decided to continue this practice we started in other parts of the code base is described in this article https://willnorris.com/2014/go-rest-apis-and-pointers/. 

The TL;DR using pointer for optional fields allow us to represent two different empty states: `uninitialized fields` and fields `intentionally set to a zero value`. This is particularly important for REST API CRUD operations where we want to be able to unset certain fields.